docs-src/sidebars.js

```diff
@@ -77,9 +69,6 @@ module.exports = {
         },
       ],
     },
-    {
-      "Project management": ["kognic-io/project"],
-    },
     {
```

Comment:
@joyuy might something in this diff have caused that two navbar items jumped to the bottom? 
![image](https://github.com/user-attachments/assets/cc7c332d-b94f-4241-a793-2c3ae7ef9466)
https://developers.kognic.com/